doc/modules/mod_jingle_sip.md

```diff
@@ -1,22 +1,22 @@
 ### Module Description
 
-This module enabled Jingle to SIP and SIP to Jingle translator.
-When this module is enalbed, MongooseIM will intercept any iq set jingle stanza with action:
+This module enables Jingle to SIP and SIP to Jingle translation.
+When this module is enabled, MongooseIM will intercept any Jingle IQ set stanza with action:
```

Comment:
"intercept" = "process and discard" or "process and route"?

Reply:
"process and discard", do you want it clarified somewhere in the text?

Reply:
I just wanted to make sure that "intercept" is used precisely here. :) If it means discarding, then probably it's intuitive enough.